cmd/kubeadm/app/cmd/phases/kubelet.go

```diff
@@ -42,6 +67,12 @@ type kubeletStartData interface {
 	KubeletDir() string
 }
 
+type kubeletStartJoinData interface {
```

Comment:
Once this is is in a separated file under `phases\join` this name can be simplified into `kubeletStartData`